### PR TITLE
update RootlessKit (1.1.0), containerd (1.6.10), Nydus (2.1.1), containerd-fuse-overlayfs (1.0.5), Kubo (0.17.0)

### PR DESCRIPTION
**Pull request by AkihiroSuda**
https://github.com/rootless-containers/rootlesskit/compare/v1.0.1...v1.1.0
https://github.com/containerd/containerd/compare/v1.6.8...v1.6.10
https://github.com/dragonflyoss/image-service/compare/v2.1.0...v2.1.1
https://github.com/containerd/fuse-overlayfs-snapshotter/compare/v1.0.4...v1.0.5

- - -
Planning to release ~v1.0.1~ v1.1.0 after merging this PR
cc @containerd/nerdctl-maintainers

### CODE REVIEW

**Review by ktock — APPROVED** [review]

**Comment by AkihiroSuda** [comment]
@containerd/nerdctl-maintainers 

I'm planning to merge this (with updates, if any) and release v1.1 next week.

**Comment by ktock** [comment]
SGTM

**Review by Zheaoli — APPROVED** [review]
LGTM

**Review by fahedouch — APPROVED** [review]

File: .github/workflows/test.yml
Comment (fahedouch) [inline .github/workflows/test.yml:69]: no need to test with `v1.7.0-beta.0` ?

Comment (AkihiroSuda) [reply .github/workflows/test.yml:69]: Substantially covered by the `main` branch.

Comment (fahedouch) [reply .github/workflows/test.yml:69]: 👌

**Pull request merged**
